lib/spack/spack/cmd/install.py

```diff
@@ -41,6 +41,8 @@
 from spack.package import PackageBase
 
 description = "build and install packages"
+section = "basic"
+level = "short"
```

Comment:
Do you want it to be possible to move commands?  I went for simple -- i.e., @citibeth's recommendation that things not move between short and long help, and the recommendation that commands only be in one section.  Is it worth it to change those restrictions?